Analysis/R/prepare_covar_cube.R

```diff
@@ -26,27 +30,28 @@ prepare_covar_cube <- function(
     res_time,
     username,
     covariate_transformations,
-    sfrac_thresh
+    sfrac_thresh,
+    sfrac_thresh_lp
```

Comment:
From what I get sfrac_thresh is used for the output shapefiles, and sfrac_thresh_lp is used for in data.

I thought that the intention was to use one value for borders, and one for connections. If this is correct, I would suggest re-naming to sfrac_thresh_border and sfrac_thresh_conn.

The next comments indicate when one and the other should  be used.